index.html

```diff
@@ -102,6 +118,104 @@ <h3 class="jobHead">software developer</h3>
           <div class="workTxt">Work</div>
         </a>
       </section>
+      <section id="projectCards">
+        <h1 class="nameHead">My recent work</h1>
+        <hr />
+        <div class="cardContainer">
+          <div class="card">
+            <div class="imgplaceholder">g</div>
```

Comment:
- [ ] It will be nice to give this class a background image or add an image to the div.

Reply:
Thanks for the feedback. Since the template only shows empty white image placeholders, I was not sure what to use as images, do i just insert a white picture? Or can i insert some random pictures i like?

Reply:
- [ ] That right. A white picture will do.